tests/stub/routing.py

```diff
@@ -115,8 +119,14 @@ def get_vars(self):
             "#ROUTINGCTX#": '{"address": "' + host + ':9001", "region": "china", "policy": "my_policy"}',
             "#EXTRA_HELLO_PROPS#": get_extra_hello_props(),
         }
-        if get_driver_name() in ['dotnet', 'java']:
+        v["#HELLO_ROUTINGCTX#"] = v["#ROUTINGCTX#"]
+
+        if get_driver_name() in ['dotnet', 'java', 'javascript']:
             v["#DBPARAM#"] = "database"
+
+        if get_driver_name() in ['javascript']:
+            v["#HELLO_ROUTINGCTX#"] = '{"region": "china", "policy": "my_policy"}'
```

Comment:
Not related to the PR, but why is this only needed on js driver?

Reply:
It's a strange behaviour. 

On the 4.x, it sends the context with the `address` when it call the procedure,  but it sends the routing context without the `address` to the HELLO message. 

On 3, it doesn't send the `address` to the procedure and the java driver is the only one to send routing context on the HELLO. 

I think the lack of address on the routing message for the 3.x could be a bug/miss-behaviour. But I don't know the impact of this.